### DISCUSSION TITLE
publish a new backend: pySigma-backend-fortisiem

### DISCUSSION THREAD

**Discussion by meiliumeiliu** [SigmaHQ/pySigma#173] [Ideas]
Hi,

Could you help me to publishing a Backend? 

https://github.com/FortiSIEM/pySigma-backend-fortisiem/tree/master

Best regards,
Mei

**Comment by thomaspatzke** [comment]
Please add it to the [plugin directory](https://github.com/SigmaHQ/pySigma-plugin-directory) in the [format described in the README](https://github.com/SigmaHQ/pySigma-plugin-directory#format).
